packages/angular_devkit/build_optimizer/src/build-optimizer/build-optimizer.ts

```diff
@@ -16,20 +16,29 @@ import { getWrapEnumsTransformer, testWrapEnums } from '../transforms/wrap-enums
 
 
 const whitelistedAngularModules = [
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)animations(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)common(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)compiler(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)core(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)forms(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)http(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)platform-browser-dynamic(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)platform-browser(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)platform-webworker-dynamic(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)platform-webworker(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)router(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)upgrade(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)material(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)cdk(\\|\/)/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]animations[\\\/]/,
```

Comment:
Nit: You don't need to escape `/` inside character groups (`[...]`).

Reply:
Fixed

Reply:
You still need to escape `\` though. So it should be `[\\/]`.
Probably a good idea to add some test too :grin:

Go Windows users! 🥂

Reply:
That's a really good idea. I should have done that ages ago. Fixed.